lebedigital/demonstrator_scripts/computation_sc_volume_fraction.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from lebedigital.unit_registry import ureg
+
+
+def computation_sc_volume_fraction(input_dic):
+    # initialize output dictionary
+    output = {}
+
+    if input_dic["sc_mass_fraction"] == 0:
+        return 0
+    else:
+        mass_sub = 1 * ureg("kg") / (1 + (1 - input_dic["sc_mass_fraction"]) / input_dic["sc_mass_fraction"])
```

Comment:
I am also not sure what I did there... I noticed I forgot to write a test (I was trusting the coverage, but without the init files, it will not check files that are not tested). I wrote a test and the results are fine. So I guess I derived some unnecessarily complex formula which can be replaced by `mass_sub = input_dic["sc_mass_fraction"] * ureg("kg")`

Reply:
I included this part in the "computation_volume_content".